src/index.js

```diff
@@ -1,11 +1,11 @@
-const { Telegraf } = require('telegraf');
-require('dotenv').config();
+const { Telegraf } = require("telegraf");
+require("dotenv").config();
 
 const bot = new Telegraf(process.env.BOT_TOKEN);
```

Comment:
**praise**: Good use of environment variables to keep sensitive information secure.